src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String REGEX = "\\D+";
+    private static final int SUPPLY_INDEX = 0;
+    private static final int BUY_INDEX = 1;
+    private static final int RESULT_INDEX = 2;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] resultCount = getCounts(fromFileName);
+        StringBuilder statistic = new StringBuilder();
+        writeToFile(
+                toFileName, String.valueOf(statistic.append(System.lineSeparator())
+                        .append("supply,")
+                        .append(resultCount[0])
+                        .append(System.lineSeparator())
+                        .append("buy,")
+                        .append(resultCount[1])
+                        .append(System.lineSeparator())
+                        .append("result,")
+                        .append(resultCount[2])
+                        .append(System.lineSeparator())));
```

Comment:
replying to your comment in commit (let's discuss in Slack such comments next time): with concatenation you risk to create a lot of new objects because String is immutable

and with StringBuilder you will create only 1 new object